Gifski/EditVideoViewController.swift

```diff
@@ -434,6 +497,14 @@ final class EditVideoViewController: NSViewController {
 		}
 		.tieToLifetime(of: self)
 
+		playerRateObserver = playerViewController.playerView.player?.observe(\.rate, options: [.new]) { [weak self] _, change in
+			if change.newValue != 0 {
```

Comment:
Updated to use `guard` and changed the logic to only show the warning once the video begins playing in reverse (instead of whenever the video starts playing and the bounce option is enabled)